dags/common/cleanup.py

```diff
@@ -1,11 +1,19 @@
+from bleach import clean
+
+
 def clean_whitespace_characters(input):
     return " ".join(input.split())
 
+
 def convert_html_subsripts_to_latex(input):
     from re import sub
 
     input = sub("<sub>(.*?)</sub>", r"$_{\1}$", input)
     input = sub("<inf>(.*?)</inf>", r"$_{\1}$", input)
-    input = sub("<sup>(.*?)</sup>", r"$^{\1}$", input)   
+    input = sub("<sup>(.*?)</sup>", r"$^{\1}$", input)
 
     return input
+
+
+def remove_specific_tags(value, tags=[], attributes=[]):
```

Comment:
Just for references why mutable arguments in function's signature are a really bad practise :) https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/   

I would suggest the following change:

```suggestion
def remove_specific_tags(value, tags=None, attributes=None):
    if tags is None:
        tags = []
    if attributes is None:
        attributes = []
```